main.js

```diff
@@ -151,6 +155,17 @@ function renderApp(app) {
   else if (app.view === 'details') {
     var detailsView = details(app.details.item)
     renderDetails.appendChild(detailsView)
+    var btn = document.getElementById('button')
+    var cartBadge = document.querySelector('span')
+    btn.addEventListener('click', function (event) {
+      app.cart.items.push(app.details.item)
+      cartBadge.textContent = '(' + (app.cart.items.length) + ')'
+      app.view = 'catalog'
+    })
+  }
+  else if (app.view === 'cart') {
```

Comment:
We'll be dealing with the cart view in a couple issues. You can remove this `if` for now.